examples/gccflags/gccflags_minimal.py

```diff
@@ -65,14 +62,28 @@ def run(self, desired_result, input, limit):
     for param, min, max in GCC_PARAMS:
       gcc_cmd += ' --param {0}={1}'.format(
         param, cfg[param])
-
-    compile_result = self.call_program(gcc_cmd)
+    return self.call_program(gcc_cmd)
+  
+  def run_precompiled(self, desired_result, input, limit, compile_result, id):
+    """
+    Run a compile_result from compile() sequentially and return performance
+    """
     assert compile_result['returncode'] == 0
 
-    run_result = self.call_program('./tmp.bin')
+    run_result = self.call_program('./tmp' + str(id) + '.bin')
     assert run_result['returncode'] == 0
+    self.call_program('rm ./tmp' + str(id) + '.bin')
```

Comment:
we should remove the temp file in a `try ... finally` block so as to not leak files.